<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASCIIArtMenu</class>
 <widget class="QMainWindow" name="ASCIIArtMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASCII Art Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnChooseImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose an image</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinSizeRatio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerateASCII">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate ASCII Art</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textResult">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>621</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>30</horstretch>
      <verstretch>30</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset to original</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinThreshValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>127.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThresh">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinEdgeValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>30.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdge">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge detect</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSpace">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add horizontal space</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioTone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tone-based</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioStructure">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Structure-based</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlur">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBlurValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContrast">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinContrastValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBright">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBrightValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIncFont">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+ </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecFont">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFontSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Font size</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
